frappe/utils/change_log.py

```diff
@@ -182,32 +184,37 @@ def parse_latest_non_beta_release(response):
 
 	return None
 
+
 def check_release_on_github(app):
-	# Check if repo remote is on github
 	from subprocess import CalledProcessError
+
 	try:
+		# Check if repo remote is on github
 		remote_url = subprocess.check_output("cd ../apps/{} && git ls-remote --get-url".format(app), shell=True).decode()
 	except CalledProcessError:
 		# Passing this since some apps may not have git initializaed in them
-		return None
+		return
 
 	if isinstance(remote_url, bytes):
 		remote_url = remote_url.decode()
 
 	if "github.com" not in remote_url:
-		return None
+		return
 
 	# Get latest version from github
 	if 'https' not in remote_url:
-		return None
+		return
```

Comment:
We probably don't need these checks now? Seems sane to remove them as it directly exits if its an SSH url, making the `is_git_url` check redundant